subsys/bluetooth/mesh/access.c

```diff
@@ -2114,10 +2139,11 @@ static void comp_data_clear(void)
 	int err;
 
 	err = settings_delete("bt/mesh/cmp");
+	err |= settings_delete("bt/mesh/cmp1");
```

Comment:
It is better to keep them separate. err1 and err2 maybe? When you see some strange value as LOG output it difficult differentiate what was initial error in case of both.